Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
By the way, if and when needed, this is also cool: https://chumper.github.io/2016/01/04/blog-setup.html.